services/spar/src/Spar/Scim/User.hs

```diff
@@ -745,19 +808,34 @@ synthesizeScimUser :: ST.ValidScimUser -> Scim.User ST.SparTag
 synthesizeScimUser info =
   let Handle userName = info ^. ST.vsuHandle
    in (Scim.empty ST.userSchemas userName (ST.ScimUserExtra (info ^. ST.vsuRichInfo)))
-        { Scim.externalId = Brig.renderValidExternalId $ info ^. ST.vsuExternalId,
+        { Scim.externalId = Brig.renderAuthAsExternalId $ info ^. ST.vsuAuthId,
           Scim.displayName = Just $ fromName (info ^. ST.vsuName),
-          Scim.active = Just . Scim.ScimBool $ info ^. ST.vsuActive
+          Scim.active = Just . Scim.ScimBool $ info ^. ST.vsuActive,
+          Scim.emails =
+            maybeToList $
+              authIdEmailWithSource (info ^. ST.vsuAuthId) >>= \ews ->
+                case ewsEmailSource ews of
+                  EmailFromExternalIdField -> Nothing
+                  EmailFromEmailField -> emailToScimEmail (ewsEmail ews)
```

Comment:
since we don't have to (and often can't) keep the data we receive unchanged, we could just add the email field here no matter where we have it from.

but this seems like the the choice that may go wrong less often.  so yes, let's keep it this way!